### PR TITLE
Feature/more file structure

### PR DESCRIPTION
**Pull request by piknikki**
## What does this PR do?
 - Trying to rebase. But ultimately, I did some css stuff and added favicon stuff.
## How should it be tested?
 - N/A -- just look at the files and I can explain if needed.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/mesh-logo.png">
+    <router-link to="/about">
+      <img class="rotating" alt="Vue logo" src="../assets/mesh-logo.png">
```
Comment (knees4bees) [inline src/views/Home.vue:4]: Nice. We'll want to update the alt text I think.

**Review by knees4bees — CHANGES_REQUESTED** [review]
This looks good, Nikki. Let's just update the alt text so it doesn't say "Vue logo" on the rotating logo.

File: src/views/Home.vue (hunk 1 above)
Comment (piknikki) [reply src/views/Home.vue:4]: You're totally right.  Will do.

**Review by knees4bees — APPROVED** [review]
Looks great now!

**Pull request merged**